common/libradoc/src/lib.rs

```diff
@@ -3,8 +3,20 @@
 
 use once_cell::sync::Lazy;
 use regex::Regex;
+use serde_generate::{rust, CodeGeneratorConfig};
+use serde_reflection::Registry;
 use std::{collections::BTreeMap, io::BufRead};
 
+pub fn quote_container_definitions(
+    registry: &Registry,
+) -> std::result::Result<BTreeMap<String, String>, Box<dyn std::error::Error>> {
+    let config = CodeGeneratorConfig::new("crate".to_string()).with_serialization(false);
+    let generator = rust::CodeGenerator::new(&config)
+        .with_derive_macros(Vec::new())
```

Comment:
Seems odd to need to pass in an empty Vec here, presumably to remove the default derives?

Reply:
That's right!

Reply:
Adding a comment